modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,164 @@
+----
+$ WORKLOAD_UPDATE_METHODS=$(oc get kv kubevirt-kubevirt-hyperconverged -n openshift-cnv -o jsonpath='{.spec.workloadUpdateStrategy.workloadUpdateMethods}')
+----
+
+. Turn off all workload update methods by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv --type json -p '[{"op":"replace","path":"/spec/workloadUpdateStrategy/workloadUpdateMethods", "value":[]}]'
+----
++
+.Example output
+[source,terminal]
+----
+hyperconverged.hco.kubevirt.io/kubevirt-hyperconverged patched
+----
+
+. Ensure that the `HyperConverged` Operator is `Upgradeable` before you continue. Enter the following command and monitor the output:
++
+[source,terminal]
+----
+$ kubectl get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.conditions"
+----
++
+.Example output
+[%collapsible]
+====
+[source,json]
+----
+[
+  {
+    "lastTransitionTime": "2022-12-09T16:29:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "ReconcileComplete"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Progressing"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T16:39:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Degraded"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Upgradeable"
+  }
+]
+----
+====
+
+. Manually update your cluster from the source EUS version to the next minor version of {product-title}:
++
+[source,terminal]
++
+====
+$ oc adm upgrade
+====
++
+.Verification
+* Check the current version by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+[NOTE]
+====
+Updating {product-title} to the next version is a prerequisite for updating {VirtProductName}. For more details, refer to the "Updating clusters" section of the {product-title} documentation.
+====
+
+. If you use the recommended *Automatic* approval strategy, {VirtProductName} automatically updates to the corresponding version after you update {product-title}. Monitor the {VirtProductName} update by running the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-cnv
+----
++
+[NOTE]
+====
+If you used the *Manual* approval strategy, you must approve the update in the web console.
+====
```

Comment:
Do we need to explain how to do the approval?

Reply:
It's one of those things where they're heavily discouraged from using the Manual approval strategy, so if they're using it, they ideally already know. (The Automatic one is the default.) I didn't want to duplicate too much info or make it seem like using Manual was in any way encouraged... I can see how this might come across as unhelpful, though. :/